tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/annotations/DeprecationAnnotationNegativeTest.java

```diff
@@ -40,11 +40,11 @@ public void testDeprecationAnnotation() {
         BAssertUtil.validateError(compileResult, i++, "constructs annotated as '@deprecated' must have 'Deprecated' " +
                 "documentation", 4, 1);
         BAssertUtil.validateError(compileResult, i++, "invalid documentation: 'Deprecated' documentation is only " +
-                "allowed on constructs annotated as '@deprecated'", 10, 1);
+                "allowed on constructs annotated as '@deprecated'", 8, 1);
```

Comment:
Consider the error msg: 
`invalid documentation: 'Deprecated' documentation is only allowed on constructs annotated as '@deprecated' `
```
# Define a type
# # Deprecated *(new warn line)*
# This type is deprecated
public type Foo CONST1|CONST2; *(old warn line)*
```